src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderSentenceAdjustment.kt

```diff
@@ -55,6 +55,9 @@ class OffenderSentenceAdjustment(
   @Column(name = "SENTENCE_SEQ", nullable = false)
   val sentenceSequence: Long,
 
```

Comment:
We don't need to map this to an actual object since we only need to know the existence of the foreign key